library/swift/src/ResponseFilterCallbacks.swift

```diff
@@ -0,0 +1,24 @@
+import Foundation
+
+public protocol ResponseFilterCallbacks {
+  /// Continue iterating through the filter chain with buffered headers and body data.
+  ///
+  /// This can only be called if the filter has previously returned `stopIteration{...}` from
+  /// `onHeaders()`/`onData()`/`onTrailers()`.
+  ///
+  /// Headers and any buffered body data will be passed to the next filter in the chain.
+  ///
+  /// If the response is not complete, this filter will still receive `onData()`/`onTrailers()`
+  /// calls.
+  func continueResponse()
+
+  /// - returns: The currently buffered data as buffered by this filter or previous ones in the
+  ///            filter chain. Nil if nothing has been buffered yet.
+  func responseBuffer() -> Data
```

Comment:
```suggestion
  func responseBuffer() -> Data?
```

Because it can be nil, right?

Reply:
Yep